py3-keyring.yaml

```diff
@@ -2,30 +2,33 @@
 package:
   name: py3-keyring
   version: 25.4.1
-  epoch: 0
+  epoch: 1
   description: Store and access your passwords safely.
   copyright:
     - license: "MIT"
   dependencies:
-    runtime:
-      - py3-jaraco.classes
-      - py3-importlib-metadata
-      - py3-importlib-resources
-      - py3-SecretStorage
```

Comment:
this was wierd. hmm. case?

Reply:
yeah it came form an auto conversion of the package. The upstream stylizes it that way for the Name but not the package file.   I'll remove the provides for this too once everything lands and no-one needs the dep any more.